src/urllib3/packages/ssl_match_hostname/__init__.py

```diff
@@ -10,10 +10,10 @@
 except ImportError:
     try:
         # Backport of the function from a pypi module
-        from backports.ssl_match_hostname import CertificateError, match_hostname
+        from backports.ssl_match_hostname import CertificateError, match_hostname  # type: ignore
```

Comment:
Ignores type in conditional imports, as discussed in https://github.com/python/mypy/issues/1297.